content/source/docs/enterprise/api/configuration-versions.html.md

```diff
@@ -60,19 +72,119 @@ curl \
 
 -> **Note**: Uploading a configuration file automatically creates a run and associates it with this configuration-version. Therefore it is unnecessary to [create a run on the workspace](./run.html#create-a-run) if a new file is uploaded.
 
-| Method | Path           |
-| :----- | :------------- |
-| POST | The upload URL is provided in the `upload-url` attribute in the `configuration-versions` resource |
+`POST https://archivist.terraform.io/v1/object/4c44d964-eba7-4dd5-ad29-1ece7b99e8da`
+
+**The URL is provided in the `upload-url` attribute in the `configuration-versions` resource.**
+
+### Request Body
+
+This POST endpoint requires the following properties as a request payload.
+
+Properties without a default value are required.
+
+| Key path | Type | Default | Description                                                                                                                                                                                                         |
+| -------- | ---- | ------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `data`   | file |         | A local .tar.gz file containing the folder of the terraform configuration files. This file can be created by running `tar -zcvf ./config.tar.gz .` from within the terraform configuration directory. |
+
+### Sample Request
+
+```shell
+curl \
+  --request PUT \
+  -F 'data=@config.tar.gz' \
```

Comment:
Recommend `--data-binary @config.tar.gz`. `-F` will send the payload as form data and fail.